src/shipit_static_analysis/shipit_static_analysis/clang/tidy.py

```diff
@@ -183,10 +183,17 @@ def list_missing_checks(self):
         else:
             logger.error('Firefox clang-tidy configuration {} should specify > 0 clang_checkers'.format(CONFIG_URL))
 
-        return [
+        # Special check -* is not listed by clang-tidy
+        required_checks = [
             check['name']
             for check in settings.clang_checkers
-            if not len(fnmatch.filter(available_checks, check['name'])) > 0
+            if check['name'] != '-*'
```

Comment:
Nit: You could just add this condition to the existing code, without defining a new `required_checks`

Reply:
Good point, it's indeed simpler